options/locale/locale_en-US.ini

```diff
@@ -176,7 +176,7 @@ string.desc = Z - A
 
 [error]
 occurred = An error occurred
-report_message = If you believe that this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
+report_message = If you believe that this is a Gitea bug, please search for issues on %[1]sGitHub%[2]s or open a new issue if necessary.
```

Comment:
Why not the whole content? `GitHub` has no translation for other languages.

Reply:
See https://github.com/go-gitea/gitea/pull/29500#issuecomment-1972314180 why this is a bad idea.